client/src/components/Modal/Modal.scss

```diff
@@ -73,7 +73,6 @@
   width: 100%;
   animation: modal--slideup .6s 1;
   background: $background-main;
-  display: block;
```

Comment:
skimmed this while looking through the Modal css, it's a benign change and small clean up.

Also, there's a `display: flex` in this same selector block, a few lines down